templates/smaily-woocommerce-admin.php

```diff
@@ -356,6 +364,98 @@ class="smaily-toggle"
 			</table>
 		</div>
 
+		<div id="checkout_subscribe">
+			<table class="form-table">
+				<tbody>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="checkbox_description">
+							<?php echo esc_html__( 'Subscription checkbox', 'smaily' ); ?>
+							</label>
+						</th>
+						<td id="checkbox_description">
+						<?php
+						esc_html_e(
+							'Customers can subscribe by checking "subscribe to newsletter" checkbox on checkout page.',
+							'smaily'
+						);
+						?>
+						</td>
+					</tr>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="checkbox_display">
+							<?php echo esc_html__( 'Display', 'smaily' ); ?>
+							</label>
+						</th>
+						<td id="smaily_checkout_display_visibility">
+							<?php
+							$checkbox_display_values = array(
+								'visible_unchecked' => __( 'Visible unchecked', 'smaily' ),
+								'visible_checked'   => __( 'Visible checked', 'smaily' ),
+								'hidden_unchecked'  => __( 'Hidden unchecked', 'smaily' ),
+
+							);
+							foreach ( $checkbox_display_values as $dp_value => $dp_translation ) {
+								$checked = $cb_display_selected === $dp_value ? 'checked' : '';
+
+								$input  = '<label for="checkbox_' . $dp_value . '">';
+								$input .= '<input type="radio" ';
+								$input .= 'id="checkbox_' . $dp_value . '" ';
+								$input .= 'name="checkbox_display" ';
+								$input .= 'value="' . $dp_value . '" ' . $checked . '>';
+								$input .= $dp_translation . '</label>';
+								echo( $input );
+							}
+							?>
```

Comment:
Wordpress PHP Coding Standards prohibits such code ([see last example](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style)).

It should be something like:
```php
<?php
$checkbox_display_values = array(
    'visible_unchecked' => __( 'Visible unchecked', 'smaily' ),
    'visible_checked'   => __( 'Visible checked', 'smaily' ),
    'hidden_unchecked'  => __( 'Hidden unchecked', 'smaily' ),

);
foreach ( $checkbox_display_values as $dp_value => $dp_translation ) : ?>
    <label for="checkbox_<?php echo $db_value; ?>">
        <input type="radio" id="checkbox_<php echo $dp_value; ?>" name="checkbox_display" value="<?php echo $dp_value; ?>"<?php $cb_display_selected === $dp_value ? ' checked' : '' ?>>
        <?php echo $dp_translation; ?>
    </label>
<?php endforeach; ?>
```

Reply:
Removed this whole block for a more simple look as suggested below. https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/45/commits/524836eba0f93c1cb6b371c34a3bab26635a8385